style.css

```diff
@@ -468,3 +794,272 @@ body {
   list-style: none;
   margin-right: 240px;
 }
+
+.form-control {
+  display: block;
+  width: 100%;
+  font-size: 1rem;
+  height: 20rem;
+  padding: 0.375rem 0.75rem;
+  border-radius: 0.3rem;
+  border: 1px solid rgb(223, 13, 13);
+  margin-top: 1rem;
+}
+
+.Programs01 {
+  display: flex;
+  flex-direction: c;
+}
+
+.meanmenu {
+  display: none;
+  flex-direction: row;
+  margin-top: 25px;
+}
+
+.headingmenu {
+  display: flex;
+  flex-direction: row;
+  justify-items: right;
+  gap: 300px;
+}
+
+.Desktop02 {
+  display: none;
+}
+
+.desktop-04 {
+  display: none;
+}
+
+.desktop_03 {
+  display: none;
+}
+
+.desktop-05 {
+  display: none;
+}
+
+.desctonic3 {
+  display: none;
+}
+
+/* ---------------------Desktop--------------------------------- */
+
+@media screen and (min-width: 780px) {
+  .humbergor-icon,
+  .section1mob {
+    display: none;
+  }
+
+  .infosection {
+    font-size: 48px;
+    line-height: 52px;
+    width: 80%;
+  }
```

Comment:
Great job creating the desktop version 🈹, but the mobile version was affected, please take into consideration the next suggestions:
- [ ] Fix the overlapping elements. 🌮
- [ ] Fix the left margin in most of the project 💯 .
- [ ] Remove the horizontal bar scroll. 📦 
- [ ] Space between elements. ⚡



https://user-images.githubusercontent.com/91301423/201500788-52b0747e-501c-4ede-9a72-eb280434d4c4.mp4